src/components/header/renderer.tsx

```diff
@@ -166,24 +171,28 @@ class Header extends React.Component<Props, State> {
       filename = Object.keys(gistData.files).find(f => gistData.files[f].language === 'JSON');
 
       if (filename === undefined) {
+        this.setState({
+          invalidUrl: true,
+        });
         throw Error();
```

Comment:
Do we still need this error? Would it make sense to show the user a bit more details about the error?

Reply:
If you are talking about "Error()" then i believe it can removed!

Reply:
Well, we need it or otherwise we continue below, no? You would need to refactor the code with else cases. Yes, please do it.

Reply:
Consider this case:
![corrected8](https://user-images.githubusercontent.com/35253743/53750754-a1c9fe00-3ed0-11e9-927b-acd577cd55ba.png)
It gets loaded even though no file named dwdwd is present. This pr resolved issues with Gist-Url. If possible can I make a separate pr for this issue sir?

Reply:
Sure